pkg/operator/ceph/object/controller_test.go

```diff
@@ -442,24 +442,29 @@ func TestCephObjectStoreController(t *testing.T) {
 						{"poolnum":9,"poolname":"my-store.rgw.buckets.data"}
 					]`, nil
 				}
+				t.Logf("returning default success for command: %s %v", command, args)
 				return "", nil
 			},
 			MockExecuteCommandWithTimeout: func(timeout time.Duration, command string, args ...string) (string, error) {
-				if args[0] == "realm" && args[1] == "list" {
-					return realmListJSON, nil
-				}
-				if args[0] == "realm" && args[1] == "get" {
-					return realmGetJSON, nil
-				}
-				if args[0] == "zonegroup" && args[1] == "get" {
-					return zoneGroupGetJSON, nil
-				}
-				if args[0] == "zone" && args[1] == "get" {
-					return zoneGetJSON, nil
-				}
-				if args[0] == "user" {
-					return userCreateJSON, nil
+				// this assumes the realm/zone/zonegroup have already been created
+				if command == "radosgw-admin" {
+					if args[0] == "realm" && args[1] == "list" {
+						return realmListJSON, nil
+					}
+					if args[0] == "realm" && args[1] == "get" {
+						return realmGetJSON, nil
+					}
+					if args[0] == "zonegroup" && args[1] == "get" {
+						return zoneGroupGetJSON, nil
+					}
+					if args[0] == "zone" && args[1] == "get" {
+						return zoneGetJSON, nil
+					}
+					if args[0] == "user" {
+						return userCreateJSON, nil
+					}
 				}
+				t.Logf("returning default success for command: %s %v", command, args)
```

Comment:
For what commands are the default success returned? There are others besides radosgw-admin?

Reply:
From the output... 

```
...
    /Users/blaine/go/src/github.com/rook/rook/pkg/operator/ceph/object/controller_test.go:445: returning default success for command: ceph [config get mon. rgw_rados_pool_pg_num_min --connect-timeout=15 --cluster=rook-ceph --conf=rook-ceph/rook-ceph.config --name=client.admin --keyring=rook-ceph/client.admin.keyring --format json]
    /Users/blaine/go/src/github.com/rook/rook/pkg/operator/ceph/object/controller_test.go:445: returning default success for command: ceph [osd pool get .rgw.root all --connect-timeout=15 --cluster=rook-ceph --conf=rook-ceph/rook-ceph.config --name=client.admin --keyring=rook-ceph/client.admin.keyring --format json]
    /Users/blaine/go/src/github.com/rook/rook/pkg/operator/ceph/object/controller_test.go:445: returning default success for command: ceph [osd pool get my-store.rgw.log all --connect-timeout=15 --cluster=rook-ceph --conf=rook-ceph/rook-ceph.config --name=client.admin --keyring=rook-ceph/client.admin.keyring --format json]
2021-09-24 16:23:46.315080 I | cephclient: setting pool property "pg_num_min" to "" on pool ".rgw.root"
2021-09-24 16:23:46.315086 I | cephclient: setting pool property "pg_num_min" to "" on pool "my-store.rgw.log"
    /Users/blaine/go/src/github.com/rook/rook/pkg/operator/ceph/object/controller_test.go:445: returning default success for command: ceph [osd pool set .rgw.root pg_num_min  --connect-timeout=15 --cluster=rook-ceph --conf=rook-ceph/rook-ceph.config --name=client.admin --keyring=rook-ceph/client.admin.keyring --format json]
...
    /Users/blaine/go/src/github.com/rook/rook/pkg/operator/ceph/object/controller_test.go:467: returning default success for command: radosgw-admin [period update --commit --rgw-realm=my-store --rgw-zonegroup=my-store --rgw-zone=my-store --cluster=rook-ceph --conf=rook-ceph/rook-ceph.config --name=client.admin --keyring=rook-ceph/client.admin.keyring]
...
    /Users/blaine/go/src/github.com/rook/rook/pkg/operator/ceph/object/controller_test.go:445: returning default success for command: ceph [config set client.rgw.my.store.a rgw_log_nonexistent_bucket true --connect-timeout=15 --cluster=rook-ceph --conf=rook-ceph/rook-ceph.config --name=client.admin --keyring=rook-ceph/client.admin.keyring --format json]
2021-09-24 16:23:46.315652 I | op-config: successfully set "client.rgw.my.store.a"="rgw_log_nonexistent_bucket"="true" option to the mon configuration database
2021-09-24 16:23:46.315656 I | op-config: setting "client.rgw.my.store.a"="rgw_log_object_name_utc"="true" option to the mon configuration database
...
    /Users/blaine/go/src/github.com/rook/rook/pkg/operator/ceph/object/controller_test.go:445: returning default success for command: ceph [dashboard get-rgw-api-access-key --connect-timeout=15 --cluster=rook-ceph --conf=rook-ceph/rook-ceph.config --name=client.admin --keyring=rook-ceph/client.admin.keyring --format json]
...
```

Also, to be clear, I have only added the log message and not changed the behavior in any other way. In general, I think it's probably a bad practice to return a default success for any commands in these tests.